PocketSocket/PSWebSocketServer.m

```diff
@@ -48,6 +48,9 @@ @interface PSWebSocketServerConnection : NSObject
 @end
 @implementation PSWebSocketServerConnection
 
+// private member variable
+bool _isV6Address = false;
```

Comment:
Also, it should be declared in PSWebSocketServer, not PSWebSocketServerConnection.
And you can't initialize an instance variable in its declaration, so the `= false` needs to be deleted.